### ISSUE TITLE
UX Discussion 1: (Map of Early Modern London, Dickinson16, Perseus, or Shelley-Godwin)

### ISSUE THREAD

**Issue by msb81** [ebeshero/DHClass-Hub#195]
I chose the Emily Dickson site to explore. What I love about this sit is that it invites you to learn more about anything you want about Emily Dickinson. It gives you a link to visit her museum and also to view her fascicles. I like this because I actually allowed you to be some what hands on with her work and see what the students have done to this webpage. I like the layout of the website. As for the UX, I think it's great. Everything is easy and accessible. The only thing that I find wrong is that I think maybe the page is a little pain. Maybe that's just me, I love bold things. But all in all, I love the UX that users get as well as how the page is set up.

**Comment by ebeshero** [comment]
@msb81 Hi Maddie! Can you tell us a little more about which page is "a little pain"? Paste a link here and point out what's problematic!

**Comment by setriplette** [comment]
I checked out the Map of Early Modern London Project. I liked the following parts of the UX:

- Site colors and typeface. I always prefer white text on black, and I liked the red and gray accents.

- Images on homepage. I liked seeing the website sections with the images of the things they were referencing.

- I could browse locations by type on the map. I'm not sure why I found the gates interesting, but it is probably because I have seen some of their names in literary texts.

- personography and placeography were tidy in appearance and easy to find

What I didn't like:

- The best information about what this site is about is in a video of the project director in which she unfortunately comes off as a wee bit pretentious. This is the only place on the site where I could learn that the whole point of this thing is to learn about Shakespeare's London.

- The map itself is totally underwhelming, especially when I've interacted with maps that work so much better. The dropdown menu on the upper left is persistent in both Firefox and Safari and in fact makes it hard to see important parts of the image. You cannot move the box or the map so that they do not overlap. The rotate map thing is silly and unnecessary. The draw line function worked and generated some kind of code to track the coordinates, but I couldn't exit out of it without shutting down the site entirely and going in from the homepage. I could find a location I had in mind (Bishopsgate), but it took me a bit to figure out that it was behind the menu. I tried the picture taking function and it didn't work as well for me as a screenshot, but then the menu blocked what I wanted to see. Very frustrating experience with the map, though I could see why it might be nice if it worked well.

- It was hard for me to perceive how people who weren't in the project team could use the site. It seems great for a class or group of researchers but it wasn't easy enough to use so that I would make an assignment from it if I taught a Renaissance class. I might show images from the map.

- I was disappointed that none of the Stow text was available in a reading view. I saw the xml files but didn't want to open them. I would have liked to have seen a page or two even if the whole thing isn't done (and don't they have a big old grant for this?)

- there is a section on the site where you can search for "texts" and it broke my heart. What I expected and wanted was mentions of the locations pulled from literary texts, OED etymology entries style. What I got was the texts on the site (encyclopedia type entries) that mentioned the locations.

- personography and placeography were no more informative than a Google search/Wikipedia search would be and were not particularly customized to the project

**Comment by PPH3** [comment]
From reading excerpts from notebook A from _Frankenstein_, I am now beginning to grasp the scope of Digital Humanities and the possibilities of this field. From the perspective of an aspiring literary scholar, I am delighted merely at the images of the original draft in its editorial stages. The handwritten pages with markup from the author is invaluable in itself. 

The companion images of the markups are a great way to see neatly typed text with the accompanying markup. You can also view the pages with the corrections. What I am most impressed with, however, is the  coded pages you can view by switching to the TEI page,

This is where the true challenge lies, as every editorial mark demands its own coding accompaniment.   
ex: <line><del rend="strikethrough"
<del rend="overwritten">at</del>
<add place="intralinear" hand="#pbs">ose</add>
This is rigorous scholarship! This group undertook a painstaking process to provide not only a  clean, modern rendition of the edits in the draft, but a coded version of these invaluable pages of classic horror. The interface, I found to be highly user-friendly. 
Shelley, M. W. "Frankenstein, Notebook A", in The Shelley-Godwin Archive, MS. Abinger c. 56, 15r. Retrieved from http://shelleygodwinarchive.org/sc/oxford/frankenstein/notebook/a/#/p33/mode/xml

This work functions as a great learning tool for literature students and aspiring coders alike. You can obtain editorial sensibilities through studying this work by observing the rudiments of this process through code, but you can also appreciate the developmental process of writing novels. All in all, I found this to be an eye-opening, yet accessible and comprehensive user experience.

edit: may come back to this tonight if time permits

**Comment by jonhoranic** [comment]
I spent an painfully long time navigating the Perseus Project site, overall I have mainly negative feelings. Looking at this as if I was just a random internet view to stumble upon this site, the first problem comes from the large amount of clashing types of content. There is a ton of text info displayed plainly in white space on the left, which is more of a visual issue then a technical one. On the right are the all to common "box within box" problem where the link is the picture itself but a basic browser wont assume its as important unless they hover over them. If you want someone to click something, let them _really_ know it. The same goes for the grey selection bar above and the facebook and twitter icons are barely noticeable.

That is all just on the home page, and I would be more comfortable with the site as a whole if the home page's issues were similar to each page that came after. Sadly this isn't the case, here is a quick rundown of glaring issues I found in order on the grey bar:

1. Collections/Texts - Each link is a new page division, a rabbit whole of navigation to get all the way down to the selections you wish to read. Key example is the link put up on Newtfire for the Homeric Hymn 7 (http://www.perseus.tufts.edu/hopper/text?doc=urn:cts:greekLit:tlg0013.tlg007.perseus-eng1), from the collections page try to navigate down to it. Imagine you had no idea what you were looking for, a lot of trial and error to find a source. What is starting on this page you wanted to work backward? It is not very clear at all.
1.1 Side note, just looking at the page with the Hymn on it is content overload, I have no idea if I clicked on any of these links where I will end up or how to get back if I was just passing through.

2. Perseus Catalog - I am suddenly in a different tab, on a different site, and have more info to navigate through. Not the smoothest transition and I don't understand the transition.

3. Research/Grants/Open Source - Super plain face, not engaging other then just being more to read through. Would like context that leads into these sections to make it more digestible.

4. About/Help - Suddenly a lighter grey bar pops up underneath the first grey bar, almost unnoticeable unless you are going to click another option on the bar. Both of these become "content dungeons" that are even deeper into sub pages of sub pages. The help section almost requires its own help section to navigate (and even though this is a joke, I wouldn't be surprised if there was one just to get my goat).

In short besides being visually stiff, it is overly complex. Pages are content dumps, information a reader would want is seated within sub pages or separate web links entirely,  the graphics are not addressed or explained as if it is assumed a reader would figure that out eventually, and the layout or pages can change dramatically withing one click.

**Comment by ahunker** [comment]
I viewed the Emily Dickinson site as well and I really enjoyed looking at it. I found it very inviting and user-friendly. I think the most inviting part of the site is how easy it is to navigate. Sometimes when I'm researching online, it's difficult to figure out where specific information is and how to access it. While looking at Emily Dickinson, I was able to determine where everything was and change from one thing to another very easily. 
 Something that I thought enhanced UX, was the way I could view the poems. I enjoyed being able to see the various versions of poems and compare them at the same time with the differences highlighted. The only thing I would change would be to able to read the poems without any highlighted texts and enjoy them as they were meant to be read, instead of just analysing the differences between the interpretations and selected manuscripts.
 I think one of the most important things I took away from this exercise and seeing how what we're doing in class can be applied in real life, especially in helping literary scholars. As a writing major and someone who enjoys reading literature, it's very useful to see how things can be outlined and presented in a digital fashion, making it more accessible to a wider range of people. I also like how  coding things, such as poems and books, introduces people to pieces of literature they might not normally read. Not only are people reading the texts, but they're studying it and analysing it, learning more about coding and literature.

**Comment by JaredKramer40** [comment]
I chose to take a look at the Map of Early Modern England and found it to be a pretty well organized site.  The images and color pallet are inviting and there are general tabs giving a good idea of what there is to come.  Showing the map gives the reader a good sense of how things were back then.  It was very neat to explore the map.  After viewing this area I felt the need to read some different documents and records  in the library and stow pages. Overall I think the UX was pretty well designed.  

Although it was a well made site there were some things that caught my eye in a negative way.  The pages with the different documents were a little overwhelming and hard to move around in. I also thought the map page should have discussed the map a little more giving better details.

It was a cool site to view overall and had a good UX in my opinion.  The map really made it a more interesting site to dig through.

**Comment by etj27** [comment]
I checked out the Pereus Progject.  Oh boy what a mess of a site.  Before I go into depth about what was wrong from my persepctive about the site I will talk about the positive things first!  There are a few aspects that I felt the site got right.

Well as far as information goes, the site does hold a great deal of textual information that can be navigated using their categorical XML system.  It invites the user to explore the Hymes and one can either read the whole document or section it off into lines so that there is sort of a bigger sense of being able to read it in a more organized fashion other than just at source value.  So there is that to consider.  

Now onto some positive aspects that I can say about the sites interface.  While there aren't many there are some.  This is so inconsequential but I did like the sites logo haha.  Personally I have always enjoyed Greek and Roman texts and mythology so reading anything that has a correlation to it usually peeks my interest in some way.  The site does attempt to separate things like any decent site should so that the user can surf through the UI to find whatever it is they came to find.  Does the site do this particular aspect well?  No, but it did try.

Now onto the effectiveness of the UX.  By the gods this site was a mess, probably one of it's worst offenses was probably the fact that there was literally no spacing between anything.  All of the sites information was literally crunched together in small text in the middle of the page.  There is a side panel on the right of every page that the user clicks onto in the table of contents if they wanted to read anything about the Hymns.  I'm not even sure what this accomplishes or how useful it is, the text is even smaller than the hymns are.  When this is put side by side next to the text in all grey it becomes visually unappealing.  This is further increased due to the color layout of the site, or lack thereof.  If I could nick name the color coding it would be, "We like white with grey boxes....WHITE WITH GREY BOXES OKAY?!"  

In the grey panel to the right there is a link to a mini map of where the text is talking about the most.  Which is fine, the map is actually very colorful but I don't think it's even necessary, if the person reading really cared about the location that the text was talking about they would just go to google maps.  I mean the map they show you on the site only leads to one place, so there is no other places on the map to give people a sense of what else is around the area, to me this somewhat defeats the purpose of having a map if there are no other references.  

So I kind mentioned how ugly the site looks above, and this makes surfing the site that much more unpleasant, since having white everything makes the site look incomplete.  There is a hymn and card interface at the top of the page which is kinda pointless.  Why have an XML format table of contents if you have a interface at the top to navigate to where someone wants to go (never mind the ugliness of it).  

I'm not sure why the Perseus catalog opens up another tab to a site.  Seems kinda pointless and somewhat confusing.  But on a positive note, there is a tutorial on how to navigate the site, which is somewhat helpful considering what I have previously stated about the site. 

When you click on Hercules on the home page there is a picture of Disney's Hercules as a reference, so I'll give that a thumbs up as a fan.  The exhibits on the home page really seem to be the only parts of the site that have more of a visual appeal.  I'm not sure why that is.  Did they just try harder for these parts or just run out of time for the others.  

But overall as far as UX goes, the biggest sin is lack of spacing of text which also falls along the separation into clearly defined categories problem.  Visual separation is arguably the most important feature of a site since it makes the user more inclined to even care about proceeding.  There are positive aspects of the site but they are overshadowed by the negatives.  By exploring the site, it doesn't really accomplish the task of discovering more about the texts.
